lmnet/tests/lmnet_tests/networks_tests/object_detection_tests/test_yolo_v2.py

```diff
@@ -687,7 +687,6 @@ def test_reorg():
     outputs = model._reorg("reorg", inputs, stride=2, data_format="NHWC", use_space_to_depth=False)
 
     tf.InteractiveSession()
-
```

Comment:
Please avoid unnecessary file change :-1:

Reply:
fixed it